scripts/style-exceptions.txt

```diff
@@ -1 +1 @@
-
+Mathlib/Data/ZMod/Basic.lean : line 1 : ERR_NUM_LIN : 1700 file contains 1505 lines, try to split it up
```

Comment:
And then another PR landed, which means you should not add to `style-exceptions.txt` anymore. Instead add `set_option linter.style.longFile 1700` to the bottom of `Data.ZMod.Basic`.

Reply:
> which means you should not add to style-exceptions.txt anymore. 

The linting script said, "run `lake exe lint-style --update` to fix this warning". So I did. Sounds like the code for `lint-style --update` needs updating...